app/lib/client.server.ts

```diff
@@ -0,0 +1,179 @@
+import { createRecord, upsertRecord, deleteRecord, getChangeStatus } from './dns.server';
+import type { RecordType } from './dns.server';
+import {
+  createRecordInDB,
+  updateRecordInDB,
+  deleteRecordInDB,
+  findRecordById,
+  doesRecordExistInDB,
+} from '../models/record.server';
+import { RecordStatus } from '@prisma/client';
+
+export const createUserDomain = async (
+  hostedZoneId: string,
+  username: string,
+  type: RecordType,
+  name: string,
+  value: string
+) => {
+  const changeId = await createRecord(hostedZoneId, type, name, value);
+
+  if (!changeId) {
+    throw new Error('Could not get ChangeID from creating a record in Route 53');
+  }
+
+  const response = await getChangeStatus(changeId);
```

Comment:
I wonder if this is useful or not?  The change will almost certainly take more than 1 call to `getChangeStatus()`, so it might be better to set it to `PENDING` by default, and to this status update in a background task?

Reply:
I thought same. I wasn't sure how this should be handled. I will remove these for now.